aspnetcore/mvc/advanced/app-parts.md

```diff
@@ -41,14 +38,14 @@ To include views in the assembly:
 * Add the following markup to the shared project file:
 
   ```csproj
-    <ItemGroup>
-      <EmbeddedResource Include = "Views\**\*.cshtml" />
-    </ ItemGroup >
+  <ItemGroup>
```

Comment:
The spacing in the content was a bit off the indentation of the whole snippet wasn't aligned to the snippet in the following bullet. That appears to be fixed when running docfx locally so I hope it's fixed in its published environment too.